src/Files.App/ViewModels/SettingsViewModels/AboutViewModel.cs

```diff
@@ -35,13 +35,10 @@ public AboutViewModel()
 			CopyAppVersionCommand = new RelayCommand(CopyAppVersion);
 			CopyWindowsVersionCommand = new RelayCommand(CopyWindowsVersion);
 			SupportUsCommand = new RelayCommand(SupportUs);
-
```

Comment:
Can you add this back?

Reply:
Thank you! In the future, please leave the conversation open and add a comment so I can easily refer back to what we were discussing.